op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -218,11 +224,19 @@ func applyStateMigrationChanges(config *genesis.DeployConfig, genesis *core.Gene
 
 	// Write the chain config to disk.
 	rawdb.WriteChainConfig(ldb, genesisHash, cfg)
-	marhslledConfig, err := json.Marshal(cfg)
+	marshalledConfig, err := json.Marshal(cfg)
 	if err != nil {
 		return nil, fmt.Errorf("failed to marshal chain config to JSON: %w", err)
 	}
-	log.Info("Wrote updated chain config", "config", string(marhslledConfig))
+	log.Info("Wrote updated chain config", "config", string(marshalledConfig))
+
+	// TODO(Alec) Is this the right place to write the genesis?
+	// Should the genesis be written before of after the cfg is updated?
```

Comment:
I guess this is the right place. My guess is that it doesn't make a difference for the genesis because the config is not part of the hashing, but we should include the hardforks.

Reply:
I have been trying but can't seem to get the setup working correct. Will update here if that changes. I haven't been able to get it working with the genesis file posted [here](https://docs.celo.org/cel2/faq) for dango either, which is why I suspect I may just be doing something wrong

Reply:
Closing this as I've fixed a couple issues that were preventing syncing (see below)